design/kanister-kopia-integration.md

```diff
@@ -599,3 +600,40 @@ credentials. This model ensures Kanister remains free from a hard dependency on
 any crypto packages, and vault-like functionalities.
 
 If misplaced, Kanister will not be able to recover these credentials.
+
+#### secrets-validation
+Approaches:
+
+1. Use `kanctl` to validate the secrets before they are created
+`kanctl` has a utility to validate resources. A new resource under `kanctl validate` command for secrets will be added. The command will also have a flag to validate what secret we need to validate.
```

Comment:
If we see rich diff, both the lines are appearing together.

Reply:
I will verify it